pluma/pluma-window-activatable.c

```diff
@@ -0,0 +1,114 @@
+/*
+ * pluma-window-activatable.h
+ * This file is part of pluma
+ *
+ * Copyright (C) 2010 Steve Frécinaux
+ *
+ *  This program is free software; you can redistribute it and/or modify
+ *  it under the terms of the GNU Library General Public License as published by
+ *  the Free Software Foundation; either version 2 of the License, or
+ *  (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Library General Public License for more details.
```

Comment:
we don't use 2 spaces between `*` and text. Test:
```
$ git grep "This program is free software;" -- ':*.c' | cut -d ':' -f 2
```

Reply:
done